conda-reqs/chipyard.yaml

```diff
@@ -46,10 +46,10 @@ dependencies:
     - findutils
     - rsync
     - psutil
-    - doit=0.35.0
+    - doit
     - gitpython
     - humanfriendly
-    - e2fsprogs
+    # - e2fsprogs
```

Comment:
Can you build buildroot without this?

Reply:
I don't think so. Will need to think of a solution once I fix the other issues with this PR.